Translated/Dialog/Bergmann/ssq_011101.csv

```diff
@@ -5,4 +5,4 @@ ssq_011101_0030#0,"""Oh, I can just imagine Gliding down from there...<br>Riding
 ssq_011101_0040#0,"""......"""
 ssq_011101_0050#0,"""Ahh, it's like climbing the old girl all over again!"""
 ssq_011101_0060#0,"""Thanks for taking the time out of your day to hook me up!"""
-ssq_011101_0070#0,"""Oh, I've got a reward for you! It might not be huge, but it all adze- I mean, adds- up!"""
+ssq_011101_0070#0,"""Oh, I've got a reward for you! It might not be huge, but please accept it!"""
```

Comment:
you monster

Reply:
You removed the pun of a man who loves climbing